pip/qsharp/_native.pyi

```diff
@@ -141,9 +141,9 @@ class Output:
     def __repr__(self) -> str: ...
     def __str__(self) -> str: ...
     def _repr_html_(self) -> str: ...
-    def state_dump(self) -> Optional[StateDump]: ...
+    def state_dump(self) -> Optional[StateDumpData]: ...
 
-class StateDump:
```

Comment:
For my own curiosity, why was the class renamed?

Reply:
This file, as explained to me by @minestarks, is `.pyi` instead of `.py` making it an interface file. So there is no implementation allowed here, it's basically just the manually declared interfaces for what comes out of Rust via PyO3. Because I wanted to add implementation, I had to create a wrapper type to expose the functionality I wanted, and renamed the native type that should stay internal.

That said, your question made me realize the editor rename incorrectly updated the name of the exposed type, which was not intentional. I'll get that fixed!